data/sql/customizations.sql

```diff
@@ -35,4 +35,20 @@ INSERT INTO authority_to_authority_preference(source_auth_name, target_auth_name
     ('IGNF', 'EPSG', 'PROJ,IGNF,EPSG' );
 
 INSERT INTO authority_to_authority_preference(source_auth_name, target_auth_name, allowed_authorities) VALUES
-    ('ESRI', 'EPSG', 'PROJ,ESRI,EPSG' );
\ No newline at end of file
+    ('ESRI', 'EPSG', 'PROJ,ESRI,EPSG' );
+
+-- Custom ellipsoids (from proj -le)
+
+INSERT INTO "ellipsoid" VALUES('PROJ','ANDREA','Andrae 1876 (Den., Iclnd.)',NULL,'PROJ','EARTH',6377104.43,'EPSG','9001',300.0,NULL,0);
+INSERT INTO "ellipsoid" VALUES('PROJ','CPM','Comm. des Poids et Mesures 1799',NULL,'PROJ','EARTH',6375738.7,'EPSG','9001',334.29,NULL,0);
```

Comment:
Similar to above: `Commission des Poids et Mesures`. Or maybe `Comité international des poids et mesures` as [French wikipedia suggests](https://fr.wikipedia.org/wiki/Comit%C3%A9_international_des_poids_et_mesures)? Final say goes to our resident Frenchman @rouault :-)

Reply:
I don't care honestly and don't want to harass Nyall with nitpicking. If it was only me, I wouldn't have included at all those esoteric and probably unused/deprecated ellipsoids...

Reply:
> probably unused/deprecated ellipsoids...

The Andræ ellipsoid is still in use for interoperability with historical registers especially for a few Icelandic systems. So not entirely unused, but certainly esoteric.